modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -168,3 +170,8 @@ interface EitherHash<L, R> : Hash<Either<L, R>>, EitherEq<L, R> {
     HR().run { it.hash() }
   })
 }
+
+@extension
+interface EitherFx<A> : Fx<EitherPartialOf<A>> {
```

Comment:
Fx is a family of type classes all with the same name and different package.
Each one of them represents a power in effectful monads:
```
arrow.typeclasses.suspended.monad.Fx
arrow.typeclasses.suspended.monaderror.Fx
arrow.effects.typeclasses.suspended.monaddefer.Fx
arrow.effects.typeclasses.suspended.concurrent.Fx
```
This allows us to only implement one of them for a given data type. For example we want the `IO` `fx` to be `arrow.effects.typeclasses.suspended.concurrent.Fx` so when users import it it already has all the concurrent combinators for races and parallelism instead of just the lower powers of `arrow.typeclasses.suspended.monad.Fx`. In other langs it may not make sense but in Kotlin this is a practical technique to divide the same type class concept which in this case is side effect suspension in different types in a hierarchy to avoid exposing lowered powered versions of it.